omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -186,12 +186,20 @@
   execute "#{rmq_plugins} enable rabbitmq_management" do
     environment (rabbitmq_env)
     user opc_username
-    not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
+    not_if { !rabbitmq['management_enabled'] || shell_out("#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E").status.success? }
```

Comment:
If `rabbitmq['management_enabled']` is a node attribute, why not just put all of that in an `if` block and leave the `not_if` the same?